### PR TITLE
Update deployments.md

### PR DESCRIPTION
**Pull request by arpitha-dhanapathi**
Added clarification around M365 GCC/GCC High/DoD offerings and corresponding Azure/Graph endpoints to use. And, update supported features with additional workloads (Teams, Security) that are available in Sovereign clouds.

### CODE REVIEW

File: concepts/deployments.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ Before calling the Microsoft Graph APIs, you should first register your applicat
 
 | National cloud | Azure AD portal endpoint| Azure AD endpoint|
 |---------------------------|----------------|----------------|
+|Azure AD (global service)|https://portal.azure.com |`https://login.microsoftonline.com`|
```
Comment (dkershaw10) [inline concepts/deployments.md:30]: Is "global service|" the official name?  Or is it "public cloud"?  What does Azure call it in their public docs?

File: concepts/deployments.md [hunk 2]
```diff
@@ -41,14 +41,18 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
 
 | National Cloud | Microsoft Graph | Graph Explorer |
 |---------------------------|----------------|----------------|
+| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 | Microsoft Graph for US Government L4 | https://graph.microsoft.us | Not supported. |
```
Comment (dkershaw10) [inline concepts/deployments.md:45]: "Not available" seems more appropriate to me

File: concepts/deployments.md [hunk 3]
```diff
@@ -41,14 +41,18 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
...
 | Microsoft Graph for US Government L5 (DOD) | https://dod-graph.microsoft.us | Not supported. |
 | Microsoft Graph Germany | https://graph.microsoft.de | Not supported. |
 | Microsoft Graph China operated by 21Vianet | https://microsoftgraph.chinacloudapi.cn | https://developer.microsoft.com/zh-cn/graph/graph-explorer-china |
-| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 
 > [!IMPORTANT]
-> If you already have an app in US Government and you're using the worldwide endpoint `https://graph.microsoft.com`, we recommend switching to the new `https://graph.microsoft.us` endpoint. Access to US Government data using worldwide endpoint is currently functional but will be disabled in near future.
+> For an app in US Government,
```
Comment (dkershaw10) [inline concepts/deployments.md:51]: > For an app in US Government, 
>   - If you are working in Office 365 GCC environment, continue using the worldwide endpoints: 'https://graph.microsoft.com' and 'https://portal.azure.com'
>   - If you are working in Office 365 GCC High environment, please use: 'https://portal.azure.us' and 'https://graph.microsoft.us'
>   - If you are working in Office 365 DoD environment, please use 'https://portal.azure.us' and 'https://dod-graph.microsoft.us'

> Access to US Government data using worldwide endpoint is currently functional but will be disabled in near future.

File: concepts/deployments.md [hunk 4]
```diff
@@ -66,7 +70,9 @@ The following Microsoft Graph features are generally available on the `/v1.0` en
 | Outlook Mail | ✔ | ✔ | ✔ |
 | Outlook Calendar | ✔ | ✔ | ✔ |
 | Personal Contacts | ✔ | ✔ | ✔ |
+| Security | ✔ | ✔ | ✔ |
```
Comment (dkershaw10) [inline concepts/deployments.md:73]: @arpitha-dhanapathi Brian should be able to generate this automatically for you based on schema deployments.  Please sync with him.  Longer term this table should be auto-generated.

File: concepts/deployments.md (hunk 1 above)
Comment (arpitha-dhanapathi) [reply concepts/deployments.md:30]: https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud - reference to global service

File: concepts/deployments.md (hunk 2 above)
Comment (arpitha-dhanapathi) [reply concepts/deployments.md:45]: From https://developer.microsoft.com/en-us/graph/graph-explorer, you could query with a .us endpoint (https://graph.microsoft.us/v1.0/me/) and it works. But .us endpoint is not actively supported in US Gov clouds.

File: concepts/deployments.md (hunk 4 above)
Comment (arpitha-dhanapathi) [reply concepts/deployments.md:73]: Yes. That's the plan. I've been in sync with Brian, and also have an epic open for automatically updating this table (as well as API level docs and Graph explorer experience) with Sovereign cloud parity data.

File: concepts/deployments.md (hunk 3 above)
Comment (arpitha-dhanapathi) [reply concepts/deployments.md:51]: @dkershaw10 - not sure of what the comment here is. Am I missing something?

Comment (dkershaw10) [reply concepts/deployments.md:51]: @arpitha-dhanapathi It's a formatting suggestion. 
Please take a look at what your MD produces on the staging server.  It's all messed up

File: concepts/deployments.md [hunk 5]
```diff
@@ -41,14 +41,19 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
...
 | Microsoft Graph for US Government L5 (DOD) | https://dod-graph.microsoft.us | Not supported. |
 | Microsoft Graph Germany | https://graph.microsoft.de | Not supported. |
 | Microsoft Graph China operated by 21Vianet | https://microsoftgraph.chinacloudapi.cn | https://developer.microsoft.com/zh-cn/graph/graph-explorer-china |
-| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 
 > [!IMPORTANT]
-> If you already have an app in US Government and you're using the worldwide endpoint `https://graph.microsoft.com`, we recommend switching to the new `https://graph.microsoft.us` endpoint. Access to US Government data using worldwide endpoint is currently functional but will be disabled in near future.
+For an app in US Government,
+•	If you are working in Office 365 GCC environment, continue using the worldwide endpoints: 'https://graph.microsoft.com' and 'https://portal.azure.com'
+•	If you are working in Office 365 GCC High environment, please use: 'https://portal.azure.us' and 'https://graph.microsoft.us'
```
Comment (davidmu1) [inline concepts/deployments.md:53]: 'please' isn't needed here.

File: concepts/deployments.md [hunk 6]
```diff
@@ -41,14 +41,19 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
...
+•	If you are working in Office 365 DoD environment, please use 'https://portal.azure.us' and 'https://dod-graph.microsoft.us'
```
Comment (davidmu1) [inline concepts/deployments.md]: 'please' isn't needed here.

File: concepts/deployments.md [hunk 7]
```diff
@@ -41,14 +41,19 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
...
 | Microsoft Graph for US Government L4 | https://graph.microsoft.us | Not supported. |
 | Microsoft Graph for US Government L5 (DOD) | https://dod-graph.microsoft.us | Not supported. |
 | Microsoft Graph Germany | https://graph.microsoft.de | Not supported. |
 | Microsoft Graph China operated by 21Vianet | https://microsoftgraph.chinacloudapi.cn | https://developer.microsoft.com/zh-cn/graph/graph-explorer-china |
-| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 
 > [!IMPORTANT]
-> If you already have an app in US Government and you're using the worldwide endpoint `https://graph.microsoft.com`, we recommend switching to the new `https://graph.microsoft.us` endpoint. Access to US Government data using worldwide endpoint is currently functional but will be disabled in near future.
+For an app in US Government,
+•	If you are working in Office 365 GCC environment, continue using the worldwide endpoints: 'https://graph.microsoft.com' and 'https://portal.azure.com'
```
Comment (davidmu1) [inline concepts/deployments.md]: If you are working in an Office 365 GCC environment,

File: concepts/deployments.md [hunk 8]
```diff
@@ -41,14 +41,20 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
...
+| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 | Microsoft Graph for US Government L4 | https://graph.microsoft.us | Not supported. |
 | Microsoft Graph for US Government L5 (DOD) | https://dod-graph.microsoft.us | Not supported. |
 | Microsoft Graph Germany | https://graph.microsoft.de | Not supported. |
 | Microsoft Graph China operated by 21Vianet | https://microsoftgraph.chinacloudapi.cn | https://developer.microsoft.com/zh-cn/graph/graph-explorer-china |
-| Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 
 > [!IMPORTANT]
-> If you already have an app in US Government and you're using the worldwide endpoint `https://graph.microsoft.com`, we recommend switching to the new `https://graph.microsoft.us` endpoint. Access to US Government data using worldwide endpoint is currently functional but will be disabled in near future.
+For an app in US Government,
```
Comment (dkershaw10) [inline concepts/deployments.md:51]: This still isn't rendering properly on the staging server
I think you need to put a greater than symbol (\>) in front of some of the lines to make sure that it's part of your important notice. 
The enumeration isn't working properly either - not sure why.
The tick marks on the URLs are wrong if you want those to appear as "code"

**Pull request merged**